### DISCUSSION TITLE
Intermittent Error - No rewards during All

### DISCUSSION THREAD

**Discussion by arunmp2004** [davetapley/helium-tax#77] [Q&A] [answered]
Recently started seeing this error from last week.
'No rewards during All' and it worked later for the same hotspot id. Is it due to helium API degradation/issues or is davetapley going through some changes?

**Comment by davetapley** [comment]
@arunmp2004 I'd _guess_ it's underlying issues, but if you are getting errors consistently (and nothing on https://status.helium.com/) then let me know more details and I'll have a look.

**Comment by davetapley** [comment, marked as answer]
@arunmp2004 I'm going to assume this was due to https://github.com/davetapley/helium-tax/issues/79, should be fixed now.

**Comment by arunmp2004** [comment]
Thank you Dave! Yes, it seems to be fixed now.

On Fri, Apr 1, 2022 at 3:42 PM Dave Tapley ***@***.***> wrote:

> @arunmp2004 <https://github.com/arunmp2004> I'm going to assume this was
> due to #79 <https://github.com/davetapley/helium-tax/issues/79>, should
> be fixed now.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/davetapley/helium-tax/discussions/77#discussioncomment-2488519>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACZELY4QYSOOTRJG7FERBOLVC5NSBANCNFSM5RZAJ3DQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Best Regards,
Arun
